miio/dreamevacuum_miot.py

```diff
@@ -0,0 +1,376 @@
+    #: int = field(metadata={'siid': 18, 'piid': 16, 'access': ['read', 'notify']})
+    # piid: 17 (): (uint16, unit: None) (acc: ['read', 'notify'], value-list: [], value-range: [0, 100, 1])
+    #: int = field(metadata={'siid': 18, 'piid': 17, 'access': ['read', 'notify']})
+    # piid: 18 (): (uint8, unit: None) (acc: ['read', 'notify'], value-list: [{'value': 0, 'description': ''}, {'value': 1, 'description': ''}], value-range: None)
+    #: int = field(metadata={'siid': 18, 'piid': 18, 'access': ['read', 'notify']})
+    # siid 19: (consumable): 3 props, 0 actions
+    # piid: 1 (life-sieve): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "life_sieve": {"siid": 19, "piid": 1},
+    # piid: 2 (life-brush-side): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "life_brush_side": {"siid": 19, "piid": 2},
+    # piid: 3 (life-brush-main): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "life_brush_main": {"siid": 19, "piid": 3},
+    # siid 20: (annoy): 3 props, 0 actions
+    # piid: 1 (enable): (bool, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "timer_enable": {"siid": 20, "piid": 1},
+    # piid: 2 (start-time): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "start_time": {"siid": 20, "piid": 2},
+    # piid: 3 (stop-time): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "stop_time": {"siid": 20, "piid": 3},
+    # siid 21: (remote): 2 props, 3 actions
+    # piid: 1 (deg): (string, unit: None) (acc: ['write'], value-list: [], value-range: None)
+    "deg": {"siid": 21, "piid": 1, "access": ["write"]},
+    # piid: 2 (speed): (string, unit: None) (acc: ['write'], value-list: [], value-range: None)
+    "speed": {"siid": 21, "piid": 2, "access": ["write"]},
+    # siid 22: (warn): 1 props, 0 actions
+    # siid 23: (map): 3 props, 1 actions
+    # piid: 1 (map-view): (string, unit: None) (acc: ['read', 'notify'], value-list: [], value-range: None)
+    "map_view": {"siid": 23, "piid": 1},
+    # piid: 2 (frame-info): (string, unit: None) (acc: ['write'], value-list: [], value-range: None)
+    "frame_info": {"siid": 23, "piid": 2},
+    # siid 24: (audio): 2 props, 3 actions
+    # piid: 1 (volume): (int32, unit: None) (acc: ['read', 'write', 'notify'], value-list: [], value-range: [0, 100, 1])
+    "volume": {"siid": 24, "piid": 1},
+    # piid: 3 (voice package): (string, unit: none) (acc: ['read', 'write'], value-list: [], value-range: None)
+    "voice_package": {"siid": 24, "piid": 3}
+    # siid 25: (): 1 props, 0 actions
+    # piid: 1 (): (string, unit: None) (acc: ['read', 'notify'], value-list: [], value-range: None)
+    # : str = field(metadata={'siid': 25, 'piid': 1, 'access': ['read', 'notify']})
+}
+
+
+class ChargingState(Enum):
+    unknown = -1
+    charging = 1
+    not_charging = 2
```

Comment:
```suggestion
    Discharging = 2
```
Please start enum members with a capitalized letter, and use camel case instead of underscore.
Also, use descriptive names (instead of what was in the json file) where possible.

This applies to other enums and their members below.